File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        dix = 1

        for i in range(1, len(nums)):
            if nums[i] != nums[i - 1]:
                nums[dix] = nums[i]
                dix += 1
        return dix
        

File: Data Structures & Algorithms/rotate-array/submission-1.java
class Solution {
    public void rotate(int[] nums, int k) {
        int l = nums.length;

        int temp = -1;
        int current = temp;

        int old;
        int count = 0;
        while (count < l - 1) {
            temp++;
            current = temp;
            old = nums[current];
            do {
                int next = (current + k) % l;
                int temp1 = nums[next];
                nums[next] = old;
                old = temp1;
                count++;
                current = next;
            } while (current != temp);
        }
    }
}